proto/viam/app/v1/app.proto

```diff
@@ -443,7 +443,6 @@ message EnableBillingServiceResponse {}
 message UpdateBillingServiceRequest {
   string org_id = 1;
   BillingAddress billing_address = 2;
-  string billing_support_email = 3;
```

Comment:
@ohEmily could you force merge this? removed this field because we realized we no longer need it (have separate email api since shared with auth service). currently unused so not really a breaking change